vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -485,13 +475,24 @@ class MessageItemFactory @Inject constructor(
             .iconRes(R.drawable.ic_paperclip)
     }
 
+    private fun buildAudioContent(
+            params: TimelineItemFactoryParams,
+            messageContent: MessageAudioContent,
+            informationData: MessageInformationData,
+            highlight: Boolean,
+            attributes: AbsMessageItem.Attributes,
+    ) = if (messageContent.voiceMessageIndicator != null) {
```

Comment:
I can see the Uncle Bob here ✨ Clean Code never mentions this in the case of null checks (and I can't find anything online about it), but imo I think `!= null` is actually positive, because it's akin to "if this data exists"